modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -1,27 +1,36 @@
 package types
 
-import "fmt"
+import (
+	"fmt"
+)
 
 const (
 	// ModuleName defines the Interchain Account module name
 	ModuleName = "interchainaccounts"
 
-	// Version defines the current version the IBC tranfer
+	// Version defines the current version the IBC interchainaccounts
 	// module supports
 	Version = "ics27-1"
 
+	// PortID is the default port id that the interchainaccounts module binds to
 	PortID = "ibcaccount"
 
-	StoreKey  = ModuleName
-	RouterKey = ModuleName
+	// StoreKey is the store key string for IBC interchainaccounts
+	StoreKey = ModuleName
 
-	// Key to store portID in our store
-	PortKey = "portID"
+	// RouterKey is the message route for IBC interchainaccounts
+	RouterKey = ModuleName
 
+	// QuerierRoute is the querier route for IBC interchainaccounts
 	QuerierRoute = ModuleName
 
-	// MemStoreKey defines the in-memory store key
-	MemStoreKey = "mem_capability"
+	// Delimiter is the delimiter used for the interchainaccounts version string
+	Delimiter = "|"
```

Comment:
I'm proposing we use a pipe `"|"` as the delimiter between ics-27 version and address as it makes for much simpler parsing/validation due to the string already being hyphenated

Reply:
I don't see any problem with this. @colin-axner